### ISSUE TITLE
Installation instructions for EL8

### ISSUE THREAD

**Issue by OleHolmNielsen** [job/aggregate6#7]
I'm grateful for the aggregate6 tool because Firewalld on EL8 (CentOS/RHEL) crashes when giving Ipsets containing overlapping CIDR ranges (see https://wiki.fysik.dtu.dk/it/Linux_firewall_configuration#using-ipsets-in-firewalld-on-rhel-centos-8).
Could you kindly add some extra Installation instructions for EL8, because the pip3 fails on a minimal EL8 system:
```
On EL8 systems you must first install some prerequisites:
$ dnf install gcc platform-python-devel
```

**Comment by Habbie** [comment]
What's the failure?

**Comment by Habbie** [comment]
Oh, do you mean, please add those two lines? If so, please use the edit button on https://github.com/job/aggregate6/blob/master/README.md and submit a PR.

**Comment by OleHolmNielsen** [comment]
I've made a PR now.  Without the prerequistes the pip3 command fails with these errors:
```
    unable to execute 'gcc': No such file or directory
    error: command 'gcc' failed with exit status 1
```
and with gcc installed also this error:
```
    gcc -pthread -Wno-unused-result -Wsign-compare -DDYNAMIC_ANNOTATIONS_ENABLED=1 -DNDEBUG -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -D_GNU_SOURCE -fPIC -fwrapv -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -D_GNU_SOURCE -fPIC -fwrapv -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -D_GNU_SOURCE -fPIC -fwrapv -fPIC -I/tmp/pip-build-t25j2n35/py-radix/radix -I/usr/include/python3.6m -c radix/_radix.c -o build/temp.linux-x86_64-3.6/radix/_radix.o
    radix/_radix.c:17:10: fatal error: Python.h: No such file or directory
     #include "Python.h"
              ^~~~~~~~~~
    compilation terminated.
    error: command 'gcc' failed with exit status 1
```

**Comment by Habbie** [comment]
Thanks, but I think you need to click another button or two to actually submit the PR - I don't see it yet.

**Comment by OleHolmNielsen** [comment]
I hope I've made the pull request correctly: https://github.com/job/aggregate6/pull/8
There's a Travis error which I don't understand at all :-(

**Comment by Habbie** [comment]
Travis error fixed in #9, we still tried to test against a discontinued Python pypy version.

#8 merged, thanks!
